### ISSUE TITLE
Combo box on AVL plugin is not operable through keyboard

### ISSUE THREAD

**Issue by claudiahub** [backstage/backstage#11251]
## Expected Behavior

Combo box on AVL plugin should be operable through keyboard.

## Actual Behavior

1. Navigate to `Home` on the left navigation. Make sure `Components` is selected on menu below the main heading.
2. Select `all` filter and in the table navigate to any application under name column with application live view plugin available (see screenshot below).
3. In the next page that opens up for selected appplication, please navigate to `Runtime resources` tab.
4. Navigate to `Resources` table and select the name of the application. A new page opens up.
5. Navigate to `Pods` table and select a link under name column.
6. In the new page that opens up, navigate to `Live view` section.
7. Select `Actuator` item from the dropdown at top of the page.
8. Verify whether `Endpoint` combobox is getting activated through keyboard or not.

![image](https://user-images.githubusercontent.com/54085737/166463454-6eebbce4-0cc4-4cea-9956-5d195aab19c0.png)

## Context

Combo box is not getting operable through keyboard.

Involved component: 
Select from '@backstage/core-components'

User Impact: When any UI component are not accessible by using keyboard then user who rely on keyboard are facing difficulty and not able to perform any action and are completely block to perform any functionality on that control element.

A11y Conformance Level: A

WCAG Success Criteria: 2.1.1

## Environment

Browser Information: 
Google Chrome v100.0.4896.127
Firefox 99.0.1
Safari 15.4
